charts/k8ssandra/values.yaml

```diff
@@ -565,13 +571,16 @@ stargate:
   tolerations: []
 reaper:
   # -- Security context override for reaper container
-  securityContext: {}
+  securityContext:
+    runAsNonRoot: true
   # -- Security context override for reaper schema init container
   schemaInitContainerConfig:
-    securityContext: {}
+    securityContext:
+      runAsNonRoot: true
   # -- Security context override for reaper config init container
   configInitContainerConfig:
```

Comment:
Interestingly there is a bug with `configInitContainerConfig`. The reaper.yaml template fails to render when it is set.   The template has:

```
{{- if (and .Values.reaper.configInitContainerConfig .Values.reaper.configInitContainerConfig.securityContext) }}
  configInitContainerConfig:
    securityContext: {{ toYaml .Values.reaper.configInitContainerConfig.securityContext | nindent 6 }}
  {{- end }}
```
but the Reaper spec does not have a `configInitContainerConfig` property. Can you fix the template and remove this property from values.yaml?

Reply:
@jsanda I believe the `configInitContainerConfig`  was used in the `reaper-operator` with reference in the types [here](https://github.com/k8ssandra/reaper-operator/blob/2a746e60e8dfb7529e6acc7ce5adf714fd063fba/api/v1alpha1/reaper_types.go).

Reply:
@jeffbanks interesting. I don't see it in [0.3.5](https://github.com/k8ssandra/reaper-operator/blob/v0.3.5/api/v1alpha1/reaper_types.go). And indeed if you set the property in values.yaml you will get a template rendering error in reaper.yaml.

Reply:
Confirmed this is a post 0.3.5 change/commit for `reaper-operator` as part of [this PR](https://github.com/k8ssandra/reaper-operator/pull/84).